SimCalorimetry/HcalSimAlgos/src/HcalSiPM.cc

```diff
@@ -147,6 +148,12 @@ void HcalSiPM::setNCells(int nCells) {
   resetSiPM();
 }
 
+void HcalSiPM::setTau(double tau) { 
+  theTau = tau;
+  if(theTau > 0) theTauInv = 1./theTau;
```

Comment:
It's also used here: https://github.com/kpedro88/cmssw/blob/99fe553e24696bdb0d494cce1b9babb38e181855/SimCalorimetry/HcalSimAlgos/src/HcalSiPM.cc#L115-L116
I prefer to keep it as a member since this is the value that's actually set in `hcalSimParameters_cfi`